hydra-filters/src/test/java/com/addthis/hydra/data/filter/bundle/TestBundleFilterTimeRange.java

```diff
@@ -13,121 +13,87 @@
-                df.format(format.parseDateTime(input).getHourOfDay()) + "" +
-                df.format(format.parseDateTime(input).getMinuteOfHour());
-        assertEquals(input, time.substring(2));
+    public void testBeforeAfterInRange() throws IOException {
+        String str = "time:TIME, before:20170101, after:20120101, timeFormat:YYYYMMDD";
+        BundleFilterTimeRange filter = Configs.decodeObject(BundleFilterTimeRange.class, str);
+        filter.filter(bundle);
+        assertTrue( filter.filter(bundle));
     }
 
     @Test
-    public void testBundleFilterTimeRangeNoTimeFormat() {
-        DateTimeZone tz = DateTimeZone.forID("EST");
-        DateTime ntzntfDateTime = utc.toDateTime(tz);
-        String input = ntzntfDateTime.toString("yyMMddHHmm");
+    public void testBeforeAfterOutRange() throws IOException {
+        String str = "time:TIME, before:1212011213, after:1301011415, defaultExit:true, " +
+                "timeFormat:yyMMddHHmm, timeZone=Australia/Brisbane";
+        BundleFilterTimeRange filter = Configs.decodeObject(BundleFilterTimeRange.class, str);
+        assertFalse(filter.filter(bundle));
+    }
 
-        BundleFilterTimeRange bftr = new BundleFilterTimeRange( timeIn, "121201", "130101", true, null, "EST");
-        DateTimeFormatter format = bftr.getTimeZoneFormat();
-        assertNull(format);
-        assertEquals("EST", bftr.getTimeZone());
+    @Test(expected=JsonMappingException.class)
+    public void testNoTime() throws IOException {
+        String str = "before:20170101, after:20120101,timeFormat:YYYYMMDD";
+        BundleFilterTimeRange filter = Configs.decodeObject(BundleFilterTimeRange.class, str);
     }
 
-    @Test
-    public void testBundleFilterTimeRangeNoTimeZoneNoTimeFormat() {
-        DateTimeZone tz = DateTimeZone.forID("EST");
-        DateTime ntzntfDateTime = utc.toDateTime(tz);
-        String input = ntzntfDateTime.toString("yyMMddHHmm");
+    @Test(expected=JsonMappingException.class)
+    public void testMinusBeforeBadTimeformat() throws IOException {
+        String str = "time:TIME, before:-1000/00";
+        BundleFilterTimeRange filter = Configs.decodeObject(BundleFilterTimeRange.class, str);
+    }
+
+    @Test(expected=JsonMappingException.class)
+    public void testNoTimeformat() throws IOException {
+        String str = "time:TIME, before:20170101, after:20120101, defaultExit:true, timeFormat:null, timeZone:EST";
+        BundleFilterTimeRange filter = Configs.decodeObject(BundleFilterTimeRange.class, str);
+    }
 
-        BundleFilterTimeRange bftr = new BundleFilterTimeRange( timeIn, "121201", "130101", true, null, null);
-        DateTimeFormatter format = bftr.getTimeZoneFormat();
-        assertNull(format);
+    @Test(expected=JsonMappingException.class)
+    public void testNoTimeformatNoTimezone() throws IOException {
+        String str = "time:TIME, before:20170101, after:20120101, defaultExit:true, timeFormat:null, timeZone:null";
```

Comment:
`timeFormat:null` and `timeZone:null` are unnecessary. Also isn't this test redundant given the previous one?